src/main/java/uk/gov/companieshouse/api/accounts/AccountsType.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.companieshouse.api.accounts;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * An enumeration of account types
+ */
+public enum AccountsType {
```

Comment:
This has been duplicated in the `document-generator-accounts` is this supposed to be duplicated?

Reply:
I believe it is supported as this is existing functionality and i copied the file from abridged  and it is working.